app/frontend/src/lib/ui/components/filterGroup.js

```diff
@@ -33,11 +38,29 @@ export const init = (groupContainerSelector, removeButtonSelector, clearButtonSe
       }
     });
   }
+
+  const mobileFiltersButton = document.getElementById(mobileFiltersButtonSelector);
+
+  if (mobileFiltersButton) {
+    mobileFiltersButton.addEventListener('click', () => {
+      Array.from(document.getElementsByClassName('filters--hide-mobile')).map((element) => showMobileFilters(element));
+    });
+  }
+};
+
+export const showMobileFilters = (element) => {
+  element.classList.toggle('filters--show-mobile');
 };
 
 export const closeAllSectionsHandler = (e) => {
   e.preventDefault();
-  Array.from(document.getElementsByClassName('govuk-accordion__section')).map((section) => section.classList.remove('govuk-accordion__section--expanded'));
+  if (e.target.innerText === 'Close all') {
```

Comment:
something doesnt feel right here as the logic seems very similar as to what you've added in the init function (and what is in the init function isnt tested) and im sure they could be consolidated in some way and hence tested. I could get a PR together to branch if you want and ill give it a bit more thought